packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -127,7 +127,7 @@ type CommonBoxVisualProps = MakeObjectResponsive<
   {
     borderRadius: keyof Border['radius'];
     borderWidth: keyof Border['width'];
-    borderColor: BorderColorString<'surface'>;
+    borderColor: BorderColorString<'surface'> | BorderColorString<'interactive'>;
```

Comment:
We support drag & drop props on Box too so I think we should support it on both.

Reply:
For now, I refactored code to avoid using `borderColor` prop. but let's allow `interactive` token for Box in separate PR because we support Drag & Drop.

Reply:
makes sense